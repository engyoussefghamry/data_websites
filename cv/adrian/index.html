<!DOCTYPE html>
<html lang="en">
    

<head>
        <title>Adrian - Portfolio Template Preview</title>
        <meta charset="utf-8">
        <meta http-equiv="X-UA-Compatible" content="IE=edge">
        <meta name="viewport" content="width=device-width, initial-scale=1">
        <meta name="description" content="Adrian is a Personal Portfolio Template">
        <meta name="keywords" content="cosmos-themes, resume, cv, portfolio, personal, developer, designer,personal resume, onepage, clean, modern">
        <meta name="author" content="Adrian Jones">


        <!-- Favicon -->
        <link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon">

        <!--Bootstrap css-->
        <link rel="stylesheet" href="css/bootstrap.css">

        <link rel="stylesheet" href="css/style.css">

        <!--Preview css-->
        <link rel="stylesheet" href="css/styles-preview.css">

        <!--Google Fonts link-->
        <link href="https://fonts.googleapis.com/css?family=Montserrat:600,700,800,900|Poppins" rel="stylesheet">

        <!-- Global site tag (gtag.js) - Google Analytics -->
        <script async src="https://www.googletagmanager.com/gtag/js?id=UA-122650090-2"></script>
        <script>
            window.dataLayer = window.dataLayer || [];
            function gtag(){dataLayer.push(arguments);}
            gtag('js', new Date());
            gtag('config', 'UA-122650090-2');
        </script>


    </head>

    <body>

        <!--Preloader Start-->
        <div id="loader-wrapper">
            <div class="loading"></div>
            <div class="loader-section section-left"></div>
            <div class="loader-section section-right"></div>
        </div>
        <!--Preloader End-->

        <!--Banner Section Start-->
        <section id="banner" class="wave-primary banner-preview" style="background-image: url(images/background/home-header-bg.jpg); height: 70vh;">
            <div class="banner-text">
                <h1 class="text-uppercase">Adrian - Template</h1>
                <!--Typed String-->
                <div id="typed-strings">
                    <p>Creative.</p>
                    <p>Clean.</p>
                    <p>Minimal.</p>
                </div>
                <span id="typed"></span>
            </div>
            <!--Creative Background Wave-->
            <svg class="wave" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 1400 100" preserveAspectRatio="none">
                <path d="M0,100c0,0,419-178,693-49.5S1400,0,1400,0v100H0z"/>
            </svg>
        </section>
        <!--Banner Section End-->

        <!--Demo Section Start-->
        <section class="demos">
            <div class="container">
                <div class="row">
                    <div class="col-md-6">
                        <div class="item">
                            <a target="_blank" href="index-main.html">
                                <img src="images/demo/demo-main.jpg" alt="">
                                <span class="demo-title">Main Demo</span>
                            </a>
                        </div>
                    </div>
                    <div class="col-md-6">
                        <div class="item">
                            <a target="_blank" href="index-videobg.html">
                                <img src="images/demo/demo-video.jpg" alt="">
                                <span class="demo-title">Video Background Demo</span>
                            </a>
                        </div>
                    </div>
                    <div class="col-md-6">
                        <div class="item">
                            <a target="_blank" href="index-particles.html">
                                <img src="images/demo/demo-particles.jpg" alt="">
                                <span class="demo-title">Particles Demo</span>
                            </a>
                        </div>
                    </div>
                    <div class="col-md-6">
                        <div class="item">
                            <a target="_blank" href="blog.html">
                            <img src="images/demo/demo-blogs.jpg" alt="">
                                <span class="demo-title">Blogs Page</span>
                            </a>
                        </div>
                    </div>
                    <div class="col-md-6 offset-md-3">
                        <div class="item">
                            <a target="_blank" href="single-blog.html">
                            <img src="images/demo/demo-single.jpg" alt="">
                                <span class="demo-title">Single Blog Page</span>
                            </a>
                        </div>
                    </div>
                </div>
            </div>
        </section>
        <!--Demo Section End-->

        <!--Footer Section Start-->
        <footer class="wave-footer">
            <div class="container">
                <div class="col-md-12">
                    <ul class="social-icons">
                        <li>
                            <a href="#">
                                <i class="fa fa-facebook"></i>
                            </a>
                        </li>
                        <li>
                            <a href="#">
                                <i class="fa fa-twitter"></i>
                            </a>
                        </li>
                        <li>
                            <a href="#">
                                <i class="fa fa-google-plus"></i>
                            </a>
                        </li>
                        <li>
                            <a href="#">
                                <i class="fa fa-dribbble"></i>
                            </a>
                        </li>
                    </ul>
                </div>
                <div class="col-md-12">
                    <!--copyright-->
                    <p>
                        Copyright &copy; 2018 Adrian Portfolio, All Right Reserved.<br>
                        Created By cosmos-themes.
                    </p>
                </div>
            </div>
            <!--Creative Background Wave-->
            <svg class="wave" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 1400 100" preserveAspectRatio="none">
                <path d="M0,100c0,0,419-178,693-49.5S1400,0,1400,0v100H0z"/>
            </svg>
        </footer>
        <!--Footer Section End-->

        <!--JQuery js-->
        <script src="js/jquery.min.js"></script>
        <!--Bootstrap js-->
        <script src="js/bootstrap.min.js"></script>
        <!--Typed js-->
        <script src="js/typed.min.js"></script>
        <!--Isotope js-->
        <script src="js/isotope.pkgd.min.js"></script>
        <!--Wow js-->
        <script src="js/wow.min.js"></script>
        <!--Magnific popup js-->
        <script src="js/jquery.magnific-popup.min.js"></script>
        <!--Site main js-->
        <script src="js/main.js"></script>
    </body>

</html>
