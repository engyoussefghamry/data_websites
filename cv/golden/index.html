<!DOCTYPE html>
<html lang="en">

<!-- Mirrored from golden-minimal.netlify.app/ by HTTrack Website Copier/3.x [XR&CO'2014], Mon, 03 Apr 2023 21:14:13 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=UTF-8" /><!-- /Added by HTTrack -->
<head>
	<meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="description" content="تصميم تجربة و واجهة المستخدم UI/UX لجميع المنصات
    ">
    <meta name="author" content="Soufian">
    <title> كولدن مينيمال | Golden Minimal</title>
    <!-- font icons -->
    <link rel="stylesheet" href="assets/vendors/themify-icons/css/themify-icons.css">
    <link rel="preconnect" href="https://fonts.googleapis.com/">
    <link rel="preconnect" href="https://fonts.gstatic.com/" crossorigin>
    <link href="https://fonts.googleapis.com/css2?family=Tajawal:wght@200;300;400;500;700&amp;display=swap" rel="stylesheet">
    <!-- Bootstrap + main styles -->
	<link rel="stylesheet" href="assets/css/steller.css">
</head>
<body data-spy="scroll" data-target=".navbar" data-offset="40" id="home">

    <!-- Page navigation    البار          -->
    <nav class="navbar navbar-expand-lg navbar-light fixed-top" data-spy="affix" data-offset-top="0">
        <div class="container">
            <a class="navbar-brand" href="#"><img src="assets/imgs/logo.svg" alt=""></a> <!-- src="here" لتغيير الصوره هنا ضع مسار اللوقو الخاص بك مكان -->
            <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
                <span class="navbar-toggler-icon"></span>
            </button>

            <div class="collapse navbar-collapse" id="navbarSupportedContent">
                <ul class="navbar-nav ml-auto align-items-center">
                    <li class="nav-item">
                        <a class="nav-link" href="#home">الرئيسية</a>
                    </li>
                    <li class="nav-item">
                        <a class="nav-link" href="#about">عني</a>
                    </li>
                    <li class="nav-item">
                        <a class="nav-link" href="#service">خدماتي</a>
                    </li>                   
                    <li class="nav-item">
                        <a class="nav-link" href="#portfolio">الأعمال</a>
                    </li>
                    <li class="nav-item">
                        <a class="nav-link" href="#blog">المدونة</a>
                    </li>
                    <li class="nav-item">
                        <a class="nav-link" href="#contact">تواصل</a>
                    </li>
                </ul>
            </div>
        </div>          
    </nav>
    <!-- End of page navibation -->

    <!-- Page Header اول قسم -->
    <header class="header" id="home">
        <div class="container">
            <div class="infos">
                <h6 class="subtitle">مرحبا،أنا</h6>
                <h6 class="title">سفيان عبدالله</h6> <!--    لتعديل الاسم       -->
                <p>مصمم ومطور UI/UX</p>                
                    <div class="buttons pt-3">
                    <button class="btn btn-primary rounded">وظفني</button>
                    <a href="assets/My%20CV.txt" download>
                    <button class="btn btn-dark rounded">نزل سيرتي</button>
                    </a>
                </div>      

                <div class="socials mt-4">
                    <!--    لتغيير روابط التواصل الاجتماعي     -->
                    <a class="social-item" href="javascript:void(0)"><i class="ti-facebook"></i></a>
                    <a class="social-item" href="javascript:void(0)"><i class="ti-linkedin""></i></a>
                    <a class="social-item" href="javascript:void(0)"><i class="ti-github"></i></a>
                    <a class="social-item" href="javascript:void(0)"><i class="ti-twitter"></i></a>
                </div>
            </div>              
            <div class="img-holder">
                <img src="assets/imgs/man.png" alt=""><!--   لتغيير الصوره ضع مسار الصوره مكان src     -->
            </div>      
        </div>  

        <!-- Header-widget قسم مشاريع و سنوات الخبره -->
        <div class="widget">
            <div class="widget-item">
                <h2>75</h2>
                <p>عميل سعيد</p>
            </div>
            <div class="widget-item">
                <h2>90</h2>
                <p>مشاريع تم إنجازها</p>
            </div>
            <div class="widget-item">
                <h2>4+</h2>
                <p>سنة خبرة</p>
            </div>
        </div>
    </header>
    <!-- End of Page Header -->
    
    <!-- About section  قسم عني -->
    <section id="about" class="section mt-3" >
        <div class="container mt-5">
            <div class="row text-center text-md-left">
                <div class="col-md-3">
                    <img src="assets/imgs/avatar.jpg" alt="" class="img-thumbnail mb-4"><!--   لتغيير الصوره ضع مسار الصوره مكان src     -->
                </div>
                <div class="pl-md-4 col-md-9">
                    <h6 class="title">سفيان عبدالله</h6>
                    <p class="subtitle">مصمم و مطور UX/UI</p>
                    <p>أنا مصمم UX / UI (iOS ، Android ، الويب) متحمس للغاية وأبحث دائمًا عن مهام جديدة ومثيرة للاهتمام.

                        أقوم بعملي بكل شغف و إبداع لتقديم التصاميم العصرية طبقاً لأساسيات تجربة واجهة المستخدم و ألتزم بإنجاز أفضل الأعمال الجذابة بصرياً .
                        
                        أسعى للتجديد و مواكبة التطور في هذا المجال دائماً لتمكين العملاء من بناء تجربة مستخدم سهلة و فعالة و مربحة حتى يتمتع مستخدمو التطبيقات و المواقع بتجربة ناجحة و ممتعة.</p>
                    <a href="assets/My%20CV.txt" download>
                        <button class="btn btn-primary rounded mt-3">نزل سيرتي</button> 
                    </a>                  
                </div>
            </div>
        </div>
    </section>

    <!-- Service section خدماتي-->
    <section id="service" class="section">
        <div class="container text-center">
            <h6 class="subtitle">ماذا أقدم</h6>
            <h6 class="section-title mb-4">خدماتي</h6>
            <p class="mb-5 pb-4">الطريق إلى التميِّز نادراً ما يكون مزدحماً</p>

            <div class="row">
                <div class="col-sm-6 col-md-3 mb-4">
                    <div class="custom-card card border">
                        <div class="card-body">
                            <i class="icon ti-crown"></i>
                            <h5>تصميم UI/UX</h5> <!--    لتغيير اسم الخدمه     -->
                        </div>
                    </div>
                </div>
                <div class="col-sm-6 col-md-3 mb-4">
                    <div class="custom-card card border">
                        <div class="card-body">
                            <i class="icon ti-desktop"></i>
                            <h5>ويب ديزاين</h5> <!--    لتغيير اسم الخدمه     -->
                        </div>
                    </div>
                </div>
                <div class="col-sm-6 col-md-3 mb-4">
                    <div class="custom-card card border">
                        <div class="card-body">
                            <i class="icon ti-mobile"></i>
                            <h5>برمجة التطبيقات</h5> <!--    لتغيير اسم الخدمه     -->
                        </div>
                    </div>
                </div>
                <div class="col-sm-6 col-md-3 mb-4">
                    <div class="custom-card card border">
                        <div class="card-body">
                            <i class="icon ti-bar-chart"></i>
                            <h5>تصميم شعارات</h5> <!--    لتغيير اسم الخدمه     -->
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>
    <!-- End of Sectoin -->

    <!-- Skills section قسم مهاراتي -->
    <section class="section">
        <div class="container text-center">
            <h6 class="subtitle">لماذا أنا</h6>
            <h6 class="section-title mb-4"> مهاراتي </h6>
                <!--        اول مهارة    -->
            <div class="row text-left">
                <div class="col-sm-6">
                    <h6 class="mb-3">الفوتوشوب</h6>
                    <div class="progress">
                        <div class="progress-bar bg-primary" role="progressbar" style="width: 89%;" aria-valuenow="25" aria-valuemin="0" aria-valuemax="100"><span>84%</span></div><!--   style="width: 89%;" لتعديل النسبة اذهب الى      -->
                    </div>
                </div>
                <!--        2 مهارة    -->
                <div class="col-sm-6">
                    <h6 class="mb-3">تصميم مواقع</h6>
                    <div class="progress">
                        <div class="progress-bar bg-primary" role="progressbar" style="width: 83%;" aria-valuenow="25" aria-valuemin="0" aria-valuemax="100"><span>83%</span></div><!--   style="width: 83%;" لتعديل النسبة اذهب الى      -->
                    </div>
                </div>
                <!--        3 مهارة    -->
                <div class="col-sm-6">
                    <h6 class="mb-3">تصميم التطبيقات</h6>
                    <div class="progress">
                        <div class="progress-bar bg-primary" role="progressbar" style="width: 95%;" aria-valuenow="25" aria-valuemin="0" aria-valuemax="100"><span>95%</span></div><!--   style="width: 95%;" لتعديل النسبة اذهب الى      -->
                    </div>
                </div>
                <!--        4 مهارة    -->
                <div class="col-sm-6">
                    <h6 class="mb-3">تصميم شعارات</h6>
                    <div class="progress">
                        <div class="progress-bar bg-primary" role="progressbar" style="width: 90%;" aria-valuenow="25" aria-valuemin="0" aria-valuemax="100"><span>90%</span></div><!--   style="width: 90%;" لتعديل النسبة اذهب الى      -->
                    </div>
                </div>
            </div>  
        </div>
    </section>
    <!-- End of Skills sections -->

    <!-- Portfolio section قسم أعمالي -->
    <section id="portfolio" class="section">
        <div class="container text-center">
            <h6 class="subtitle">أعمالي</h6>
            <h6 class="section-title mb-4">ألق نظرة على بعض من أعمالي</h6>

            <div class="row">
                <div class="col-sm-4">
                    <div class="img-wrapper">
                        <img src="assets/imgs/folio-1.jpg" alt=""> <!--src لتغيير الصوره ضع مسار الصوره مكان -->
                        <div class="overlay">
                            <div class="overlay-infos">
                                <h5>تصميم التطبيقات</h5>   <!--    لتغيير اسم العمل     -->
                                <a href="javascript:void(0)"><i class="ti-zoom-in"></i></a>
                                <a href="javascript:void(0)"><i class="ti-link"></i></a>
                            </div>  
                        </div>
                    </div>
                    <div class="img-wrapper">
                        <img src="assets/imgs/folio-2.jpg" alt=""> <!--لتغيير الصوره ضع مسار الصوره مكان src-->
                        <div class="overlay">
                            <div class="overlay-infos">
                                <h5>تصميم مواقع</h5><!--    لتغيير اسم العمل     -->
                                <a href="javascript:void(0)"><i class="ti-zoom-in"></i></a>
                                <a href="javascript:void(0)"><i class="ti-link"></i></a>
                            </div>                              
                        </div>
                    </div>                  
                </div>
                <div class="col-sm-4">
                    <div class="img-wrapper">
                        <img src="assets/imgs/folio-3.jpg" alt=""> <!--لتغيير الصوره ضع مسار الصوره مكان src-->
                        <div class="overlay">
                            <div class="overlay-infos">
                                <h5>الفوتوشوب</h5><!--    لتغيير اسم العمل     -->
                                <a href="javascript:void(0)"><i class="ti-zoom-in"></i></a>
                                <a href="javascript:void(0)"><i class="ti-link"></i></a>
                            </div>  
                        </div>
                    </div>
                    <div class="img-wrapper">
                        <img src="assets/imgs/folio-4.jpg" alt=""><!--لتغيير الصوره ضع مسار الصوره مكان src-->
                        <div class="overlay">
                            <div class="overlay-infos">
                                <h5>تصميم شعارات</h5><!--    لتغيير اسم العمل     -->
                                <a href="javascript:void(0)"><i class="ti-zoom-in"></i></a>
                                <a href="javascript:void(0)"><i class="ti-link"></i></a>
                            </div>                              
                        </div>
                    </div>                  
                </div>
                <div class="col-sm-4">
                    <div class="img-wrapper">
                        <img src="assets/imgs/folio-5.jpg" alt=""> <!--لتغيير الصوره ضع مسار الصوره مكان src-->
                        <div class="overlay">
                            <div class="overlay-infos">
                                <h5>تصميم مواقع</h5><!--    لتغيير اسم العمل     -->
                                <a href="javascript:void(0)"><i class="ti-zoom-in"></i></a>
                                <a href="javascript:void(0)"><i class="ti-link"></i></a>
                            </div>  
                        </div>
                    </div>
                    <div class="img-wrapper">
                        <img src="assets/imgs/folio-6.jpg" alt=""> <!--لتغيير الصوره ضع مسار الصوره مكان src-->
                        <div class="overlay">
                            <div class="overlay-infos">
                                <h5>UX تصميم</h5><!--    لتغيير اسم العمل     -->
                                <a href="javascript:void(0)"><i class="ti-zoom-in"></i></a>
                                <a href="javascript:void(0)"><i class="ti-link"></i></a>
                            </div>                              
                        </div>
                    </div>                  
                </div>
            </div>

        </div>
    </section>
    <!-- End of portfolio section -->


    <!-- Blog Section قسم مدونتي-->
    <section id="blog" class="section">
        <div class="container text-center">
            <h6 class="subtitle">مدونتي</h6>
            <h6 class="section-title mb-4">آخر المقالات</h6>

            <div class="row text-left">
                <div class="col-md-4">
                    <div class="card border mb-4">
                        <img src="assets/imgs/blog-1.jpg" alt="" class="card-img-top w-100"> <!--لتغيير الصوره ضع مسار الصوره مكان src-->
                        <div class="card-body">
                            <h5 class="card-title">اهمية الموقع الشخصى للمصممين والمطورين</h5><!--    لتغيير العنوان     -->
                            <div class="post-details">
                                <a href="javascript:void(0)">بقلم: كاتب</a>
                                <a href="javascript:void(0)"><i class="ti-thumb-up"></i> 456</a>
                                <a href="javascript:void(0)"><i class="ti-comment"></i> 123</a>
                            </div>
                            <p>لماذا يجب ان يمتلك على المبرمج او المصمم ان يمتلك موقع شخصى للحصول عملاء او حتى وظيفة</p>
                            <a href="javascript:void(0)">إقرأ المزيد..</a>
                        </div>
                    </div>
                </div>
                <div class="col-md-4">
                    <div class="card border mb-4">
                        <img src="assets/imgs/blog-2.jpg" alt="" class="card-img-top w-100"> <!--لتغيير الصوره ضع مسار الصوره مكان src-->
                        <div class="card-body">
                            <h5 class="card-title">الفرق بين php و html</h5><!--    لتغيير العنوان     -->
                            <div class="post-details">
                                <a href="javascript:void(0)">بقلم: كاتب</a>
                                <a href="javascript:void(0)"><i class="ti-thumb-up"></i> 456</a>
                                <a href="javascript:void(0)"><i class="ti-comment"></i> 123</a>
                            </div>
                            <p>اذا كنت مبتدئ فى مجال الويب ربما سمعت باللغتين الاشهر على الاطلاق فى هذا المجال وهما HTML و PHP ويحيرك الفرق بين php و html.</p>
                            <a href="javascript:void(0)">إقرأ المزيد..</a>
                        </div>
                    </div>
                </div>
                <div class="col-md-4">
                    <div class="card border mb-4">
                        <img src="assets/imgs/blog-3.jpg" alt="" class="card-img-top w-100"> <!--لتغيير الصوره ضع مسار الصوره مكان src-->
                        <div class="card-body">
                            <h5 class="card-title">ماهى جافاسكربت Javascript وكيف تعمل؟</h5><!--    لتغيير العنوان     -->
                            <div class="post-details">
                                <a href="javascript:void(0)">بقلم: كاتب</a>
                                <a href="javascript:void(0)"><i class="ti-thumb-up"></i> 456</a>
                                <a href="javascript:void(0)"><i class="ti-comment"></i> 123</a>
                            </div>
                            <p>جافاسكربت من اللغات المستخدمة فى تطوير الويب بشكل واسع هذه اللغة تستخدم لتجعل صفحة الويب </p>
                            <a href="javascript:void(0)">إقرأ المزيد..</a>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

    <!-- Hire me section قسم وظفني-->
    <section class="bg-gray p-0 section">
        <div class="container">
            <div class="card bg-primary">
                <div class="card-body text-light">
                    <div class="row align-items-center">
                        <div class="col-sm-9 text-center text-sm-left">
                            <h5 class="mt-3">قم بتوظيفي لمشروعك</h5>
                            <p class="mb-3">سعادة العميل هدفنا،نسعى دائما لتقديم الأفضل</p>
                        </div>
                        <div class="col-sm-3 text-center text-sm-right">
                            <a href="#" target="_blank">  <!--href=""رابط خدمتك على خمسات او مستقل-->
                            <button class="btn btn-light rounded">وظفني!</button>
                            </a>
                        </div>
                    </div>
                </div> 
            </div>
        </div>
    </section>      
    <!-- End od Hire me section. -->

    <!-- Contact Section  قسم التواصل -->
    <section id="contact" class="position-relative section">
        <div class="container text-center">
            <h6 class="subtitle">للإستفسار او البحث عن مساعدة تواصل معنا
            </h6>
            <h6 class="section-title mb-4"> تواصل</h6>
            
            <div class="contact text-left">
                <div class="form">
                    <h6 class="subtitle">متوفر 24/7</h6>
                    <h6 class="section-title mb-4">تواصل</h6>
                    <!--  رابط شرح لربط ناجح https://youtu.be/RkDsxoyyPS4  | ضع ايمايلك  /your@email.com -->
                    <form action="https://formsubmit.co/your@email.com" method="POST">
                        <div class="form-group">
                            <input type="email" class="form-control" id="exampleInputEmail1" aria-describedby="emailHelp" placeholder="بريدك الالكتروني" required>
                        </div>
                        <div class="form-group">
                            <input type="password" class="form-control" id="exampleInputPassword1" placeholder="موضوع الرسالة" required>
                        </div>
                        <div class="form-group">
                            <textarea name="contact-message" id="" cols="30" rows="5" class="form-control" placeholder="الرسالة"></textarea>
                        </div>
                        <button type="submit" class="btn btn-primary btn-block rounded w-lg">إرسال</button>
                    </form>
                </div>
                <div class="contact-infos">
                    <div class="item">
                        <i class="ti-location-pin"></i>
                        <div class="">
                            <h5>العنوان:</h5>
                            <p> 113 Street - طريق الوحدة - الشارقة</p>
                        </div>                          
                    </div>
                    <div class="item">
                        <i class="ti-mobile"></i>
                        <div>
                            <h5>الهاتف:</h5>
                            <p>(123) 456-7890</p>
                        </div>                          
                    </div>
                    <div class="item">
                        <i class="ti-email"></i>
                        <div class="mb-0">
                            <h5>بريد إلكتروني:</h5>
                            <p>info@website.com</p>
                        </div>
                    </div>
                </div>                  
            </div>
        </div>  
        <div id="map">
            <iframe src="https://snazzymaps.com/embed/61257"></iframe>
        </div>      
    </section>
    <!-- End of Contact Section -->

    <!-- Page Footer -->
    <footer class="page-footer">
        <div class="container">
            <div class="row align-items-center">
                <div class="col-sm-6">
                    <p>كل الحقوق محفوظة <script>document.write(new Date().getFullYear())</script> &copy; <a href="http://digiweb.cf/" target="_blank">Soufian</a></p>
                </div>
                <div class="col-sm-6">
                    <div class="socials">
                        <!--    لتغيير روابط التواصل الاجتماعي     -->
                        <a class="social-item" href="javascript:void(0)"><i class="ti-facebook"></i></a>
                        <a class="social-item" href="javascript:void(0)"><i class="ti-linkedin""></i></a>
                        <a class="social-item" href="javascript:void(0)"><i class="ti-github"></i></a>
                        <a class="social-item" href="javascript:void(0)"><i class="ti-twitter"></i></a>
                    </div>
                </div>
            </div>
        </div>
    </footer> 
    <!-- End of page footer -->
	
	<!-- core  -->
    <script src="assets/vendors/jquery/jquery-3.4.1.js"></script>
    <script src="assets/vendors/bootstrap/bootstrap.bundle.js"></script>
    <!-- bootstrap 3 affix -->
	<script src="assets/vendors/bootstrap/bootstrap.affix.js"></script>

    <!-- steller js -->
    <script src="assets/js/steller.js"></script>

</body>

<!-- Mirrored from golden-minimal.netlify.app/ by HTTrack Website Copier/3.x [XR&CO'2014], Mon, 03 Apr 2023 21:14:22 GMT -->
</html>
